packages/paste-core/components/date-picker/stories/index.stories.tsx

```diff
@@ -0,0 +1,298 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Label} from '@twilio-paste/label';
+import {Text} from '@twilio-paste/text';
+import {Anchor} from '@twilio-paste/anchor';
+import {HelpText} from '@twilio-paste/help-text';
+import {useUID} from '@twilio-paste/uid-library';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {Combobox} from '@twilio-paste/combobox';
+import {DatePicker, returnDateFormatter} from '../src';
+
+// Write date picker stories and export to use in test file
+// Stories: default, one using onChange with formatter function (copy combobox controlled using state story) and render value, replicate input stories
+
+export const DefaultDatePicker: React.FC = (props) => {
+  const uidDP = useUID();
+  const uidHT = useUID();
+  return (
+    <>
+      <Label htmlFor={uidDP}>Label</Label>
+      <DatePicker id={uidDP} aria-describedby={uidHT} {...props} />
+      <HelpText id={uidHT}>Help text</HelpText>
```

Comment:
Let's make these examples a little more realistic because we have plans to use stories in Tests, the doc site, and more in the future.
i.e.: HelpText shouldn't be "help text" :P